ui/src/DebugReporter.ts

```diff
@@ -9,6 +9,20 @@ import { IInternalTelemetryReporter } from './createTelemetryReporter';
 export class DebugReporter implements IInternalTelemetryReporter {
     constructor(private _extensionName: string, private _extensionVersion: string, private _verbose: boolean) { }
 
+    /**
+     * Implements `postEvent` for `IExperimentationTelemetry`.
+     */
+    public postEvent(): void {
+        // Not needed
+    }
+
+    /**
+     * Implements `setSharedProperty` for `IExperimentationTelemetry`
+     */
+    public setSharedProperty(): void {
+        // Not needed
```

Comment:
There's not a good way to verify that the telemetry is actually working with this in place.